### ISSUE TITLE
ExcelColumnName  兼容[Description]或[Display]特性

### ISSUE THREAD

**Issue by niltor** [mini-software/MiniExcel#674]
### Excel Type

- [ ] XLSX

### Description

基础类库中已经存在`Description`及`Display`等特性,其含义表示描述或显示内容

为了更加通用,以及减少特性数量, 希望可`ExcelColumnName`的效果可以同时支持如`Description`或`Display`,
`ExcelColumnName`可以作为最高优先级.

**Comment by michelebastione** [comment]
I disagree. First of all, those attributes aren't really meant to be used for serialization. Secondly, I think that allowing multiple attributes to have the same effect on column names comes with little to no benefit while potentially causing confusion and unintended behaviours. Let's assume for example that `DisplayAttribute` was added to a property to show a special placeholder on a UI text field, now suddenly when you export your Excel file your column names are wrong and you need to find a workaround for that, probably having to use more attributes anyways. 
On the other hand, having a single specific attribute from the library that does that is unambiguous and comes with no surprises.
